tests/tests_relp.yml

```diff
@@ -113,6 +110,21 @@
       register: __result
       failed_when: __result.stdout != "1"
 
+    - name: Check tls.permittedpeer in relp_server0
+      shell: >-
+        grep 'tls.permittedpeer=\["\*.client.com","\*.example.com"\]'
+        "{{ __test_relp_server0 }}" | wc -l
+      register: __result
+      failed_when: __result.stdout != "1"
+
+    - name: Check tls.permittedpeer in relp_server1
+      shell: >-
+        grep "tls.permittedpeer=\[\"\*.$( _domain=$(hostname -d);
+        _domain=${_domain:-"localdomain"}; echo $_domain )\"\]"
```

Comment:
Can you use the value of `logging_domain` here?  If not - is the problem that the `logging_domain` value is not exported from the role, for use in testing?  If so, you can use the `public: true` option of `include_role` to export the vars and defaults - https://docs.ansible.com/ansible/2.9/modules/include_role_module.html#include-role-module

Reply:
Thank you, @richm. It worked!